.cico/cico_openshift_e2e.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Copyright (c) 2012-2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+set -ex
+
+# Setup to find nessasary data from cluster setup
+export CI="openshift"
+
+# Overrideable information
+export DEFAULT_INSTALLER_ASSETS_DIR=${DEFAULT_INSTALLER_ASSETS_DIR:-$(pwd)}
+export KUBEADMIN_USER=${KUBEADMIN_USER:-"kubeadmin"}
+export KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeadmin-password"}
+
+# Exported to current env
+export KUBECONFIG=${KUBECONFIG:-"${DEFAULT_INSTALLER_ASSETS_DIR}/auth/kubeconfig"}
+
+# Check if file with kubeadmin password exist
+if [ ! -f $KUBEADMIN_PASSWORD_FILE ]; then
+    echo "Could not find kubeadmin password file"
+    exit 1
+fi
+
+if [ ! -f $KUBECONFIG ]; then
```

Comment:
Why kubeconfig is required? To grab the default cluster URL and TLS configuration?

Maybe it's better to not to check kubeconfig file existence and instead do `oc login -u $KUBEADMIN_USER -p $KUBEADMIN_PASSWORD $OPENSHIFT_CLUSTER` where OpenShift Cluster is `https://api.crc.testing:6443 ` in case of crc. < then `--insecure-skip-tls-verify` probably should be specified or CA file must be configured.